src/routes/docs/tooling/command-center/keyboard-shortcuts/+page.markdoc

```diff
@@ -0,0 +1,64 @@
+---
+layout: article
+title: Keyboard Shortcuts
+description: Learn to navigate the Appwrite Console efficiently and effectively with your keyboard
+---
+
+The Appwrite Console was designed with a keyboard first approach. Developer experience is greatly improved by virtue of the keyboard's ease of use, quick and seamless execution of essential actions and the consistency of shortcut keys to traverse the Appwrite Console.
+
+# Shortcuts {% #shortcuts %}
+
+The Appwrite Console supports keyboard shortcuts that make it easier to navigate and perform common actions quicker. The shortcuts use the following pattern: use the first letter from the call to action followed by the resource, product, service, or page you're targeting.  For example, the shortcut keys `G` + `S` navigates to the project's Storage screen. Similarly, the shortcut `G` + `F` navigates to the project's Functions screen. However, there are some exceptions. The shortcut `G` + `E` navigates to project Settings, because `G` + `S` is in use for Storage. This pattern is used to make usage of shortcuts consistent and predictable.
```

Comment:
Please remember to remove double spaces.

> However, there are some exceptions. The shortcut `G` + `E` navigates to project Settings, because `G` + `S` is in use for Storage. 

It would be good to reword this to explain a bit more. When there's a conflict with a shortcut key, the following letter is used. For example, since S is already for Storage, the shortcut for Settings is E.